README.md

```diff
@@ -77,43 +79,43 @@ output "command-to-generate-gcp-cred-config" {
-| ------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------------- | -------- | ---------------- |
-| gcp_region                | The GCP project region where planning to create resources.                                                         | `string`      |          |`us-east-1`      |
-| gcp_project_id            | The GCP project id where you want to create resources.                                                             | `string`      | Yes      |                 |
-| gcp_project_number        | The GCP project number of above passed project id.                                                                 | `number`      | Yes      |              |
-| is_service_account_exists | Set this to false                                                                                                  | `bool`        |          | `false`          |
-| service_account_name      | The GCP service account name                                                                                       | `string`      |          | `"sa-for-uptycs"` |
-| host_aws_account_id       | Uptycs's AWS Account ID. Copy from Uptycs's GCP Integration Screen UI                                              | `number`      | Yes      |              |
-| host_aws_instance_role    | Uptycs's AWS Role name. Copy from Uptycs's GCP Integration Screen UI                                               | `string`      | Yes      |              |
-| gcp_workload_identity     | Workload Identity Pool to allow Uptycs integration via AWS federation                                              | `string`      |          | `"wip-uptycs"`             |
-| gcp_wip_provider_id       | Workload Identity Pool provider id allow to add cloud provider                                                     | `string`      |          | `"uptycs-aws-idp"`             |
 
+| Name                      | Description                                                           | Type           | Required | Default            |
+| --------------------------- | ----------------------------------------------------------------------- | ---------------- | ---------- | -------------------- |
+| gcp_region                | The GCP project region where planning to create resources.            | `string`       |          | `us-east-1`        |
+| gcp_project_id            | The GCP project id where you want to create resources.                | `string`       | Yes      |                    |
+| gcp_project_number        | The GCP project number of above passed project id.                    | `string`       | Yes      |                    |
+| is_service_account_exists | Set this to false                                                     | `bool`         |          | `false`            |
```

Comment:
`is_service_account_exists` is another badly named variable.
should be renamed to `service_account_exists`.

Can we change these and take a one time hit?